static/src/stylesheets/module/commercial/_advert.scss

```diff
@@ -115,6 +115,18 @@
     }
 }
 
+.advert__kicker {
+    &:after {
```

Comment:
Ah good to know - just [read up on this](http://stackoverflow.com/a/11701951) - fairly subtle!